Implementing COUNTER mode for saga injectors (#24)
react-boilerplate/redux-injectors: docs/api.md

-   `DAEMON` **[String][26]** Causes the saga to be started on component instantiation and never canceled
    or started again.
-   `ONCE_TILL_UNMOUNT` **[String][26]** Behaves like 'RESTART_ON_REMOUNT' but never runs it again.

> -   `COUNTER` **[String][26]** The saga will be mounted similar to 'RESTART_ON_REMOUNT', only difference is that
>     saga will be mounted only once on first inject, and ejected when all injectors are unmounted.
>     So this enables you to have multiple injectors with same saga and key, only one instance of saga will run
>     and enables you to have system that are more similar to widgets

Reviewer: ```suggestion
-   `COUNTER` **[String][26]** Similar to 'RESTART_ON_REMOUNT' except the
    saga will be mounted only once on first inject and ejected when all injectors are unmounted.
    This enables you to have multiple injectors with the same saga and key and only one instance of the saga will run
```

Author: Thank you for help, i have updated as u suggested
now its much better!
